lib/id3c/cli/io/adapters.py

```diff
@@ -0,0 +1,50 @@
+"""
+Utilities for interfacing with various systems.
+"""
+from googleapiclient import discovery
+from google.oauth2 import service_account
+from typing import Tuple
+from os import environ
+from hashlib import sha1
+import pandas as pd
+import json
+
+GOOGLE_SERVICE_ACCOUNT_CREDS_ENV_VAR = "GOOGLE_SERVICE_ACCOUNT_CREDENTIALS_FILE_PATH"
+GOOGLE_API_SCOPE_READ_ONLY = "https://www.googleapis.com/auth/spreadsheets.readonly"
+
+def read_google_sheet_as_dataframe(document_id: str,
+                sheet_range: str
+                ) -> Tuple[pd.DataFrame, str] :
+    """
+    Reads the values of a sheet of a Google spreadsheet
+    and returns a dataframe populated with the data. All columns
+    in the dataframe are strings.
+
+    Requires the GOOGLE_SERVICE_ACCOUNT_CREDS_ENV_VAR environment variable be
+    set with the path to the Google service account credentials file. Also requires
+    that the service account be granted read access to the document being read.
+    """
+
+    scopes = [GOOGLE_API_SCOPE_READ_ONLY]
+    credentials = service_account.Credentials.from_service_account_file(environ[GOOGLE_SERVICE_ACCOUNT_CREDS_ENV_VAR], scopes=scopes)
+    service_resource = discovery.build('sheets', 'v4', credentials=credentials)
+    sheet_service_resource = service_resource.spreadsheets()
+    sheet_contents_dict = sheet_service_resource.values().get(spreadsheetId=document_id, range=sheet_range).execute()
+    sheet_values = sheet_contents_dict.get('values', [])
+
+    # Convert the dictionary returned to json in order to create a
+    # digest representing the data as it was returned by Google.
+    contents_json = json.dumps(sheet_contents_dict)
+    digest = sha1(contents_json.encode('utf-16')).hexdigest()
+
+    # Use values from the first row as column names.
+    # Sometimes users populate columns that have no header/title.
+    # Add unique dummy column names for those.
```

Comment:
This isn't really a code suggestion as much as a documentation suggestion -- 
Recently the Blab dev team has been working toward being more inclusive in the language we choose to write in our programs. @tsibley pointed me to [Google's inclusive documentation writing guide](https://developers.google.com/style/inclusive-documentation) which I've found super informative. Would you consider replacing the word "dummy" here with something else? Thanks!

Reply:
Thanks a lot for pointing me to this doc, Kairsten. The new implementation doesn't include this code, so I'm resolving this thread.